Added Light Mode

AmanRaj1608 opened the pull request:
Fixes #11 

### Toggle button added for light mode

Default mode is dark mode and there is a button for light mode

###  Screenshot Dark Mode:
![image](https://user-images.githubusercontent.com/42104907/56853075-96c7a300-6940-11e9-932c-c2627fb3739c.png)

### Screenshot Light Mode:

![image](https://user-images.githubusercontent.com/42104907/56853090-c2e32400-6940-11e9-8264-7c0866612f47.png)

###  Zoomed Dark button:

![image](https://user-images.githubusercontent.com/42104907/56853918-19a22b00-694c-11e9-8588-6668721c9a20.png)

###  Zoomed Light button:

![image](https://user-images.githubusercontent.com/42104907/56853939-55d58b80-694c-11e9-8712-50dd8b44f588.png)


### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)


### Non Functional Requirement
- [x] Follows the code style of this project.
- [ ] Tests Cover Changes
- [x] I have performed a self-review of my own code
- [ ] All new and existing tests passed.
- [ ] Documentation

<!-- Thanks For your PR -->

AmanRaj1608 commented on public/darkmode.js:
```diff
@@ -0,0 +1,11 @@
+$('#toggleTheme').on('change', () => {
```
created separate js file - darkmode.js

AmanRaj1608 commented on public/style.css:
```diff
@@ -90,4 +91,102 @@ footer > a {
+.toggleTheme input[type=checkbox]:checked + .toggle-box-label-left + .toggle-box-label {
+  color: rgba(250, 250, 250, 0.911);
+  font-weight: bold;
+}
```
Made all css changes to same css file

AmanRaj1608 commented:
@aashutoshrathi 
Please review and suggest changes

aashutoshrathi commented on views/index.html:
```diff
@@ -24,8 +24,14 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.0.0-rc.16/js/uikit.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.0.0-rc.16/js/uikit-icons.min.js"></script>
     <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="../public/style.css">
```
Please remove this line it's useless.

AmanRaj1608 replied:
True 😅

aashutoshrathi commented on views/index.html:
```diff
@@ -76,6 +82,6 @@
     <!-- Place this tag in your head or just before your close body tag. -->
     <script async defer src="https://buttons.github.io/buttons.js"></script>
-
+    <script src="../public/darkmode.js"></script>
```
```suggestion
<script src="/darkmode.js"></script>
```
and place it below 
```js
<script src="/client.js"></script>
```

AmanRaj1608 replied:
Done

aashutoshrathi commented on public/darkmode.js:
```diff
@@ -0,0 +1,11 @@
+    if ($('#toggleTheme').is(":checked")) {
+        $('body').removeClass('uk-section-secondary')
+        $('body').addClass('uk-section-defaultd')
+        console.log('>>> [darkmode.js:6] light ')
```
I think we don't need logging here anymore. and also on line 9.

AmanRaj1608 replied:
Yes removed it

AmanRaj1608 commented:
@aashutoshrathi 
Added all changes and fixed lint issue 😄

aashutoshrathi commented on public/darkmode.js:
```diff
@@ -0,0 +1,9 @@
+$('#toggleTheme').on('change', () => {
+  if ($('#toggleTheme').is(':checked')) {
+    $('body').removeClass('uk-section-secondary');
+    $('body').addClass('uk-section-defaultd');
```
I think it should be `uk-section-default` in both places since there is no such class as `uk-section-defaultd`

rossoskull commented on views/index.html:
```diff
@@ -26,6 +26,11 @@
     <link rel="stylesheet" href="/style.css">
   </head>
   <body class="uk-section uk-section-secondary uk-margin-remove">
+      <span class="toggleTheme">
```
Incorrect indentation.

rossoskull commented:
You can take reference from https://codepen.io/mallendeo/pen/eLIiG for some cool toggle buttons.

AmanRaj1608 commented:
Please review again @aashutoshrathi @rossoskull 

1. Added Flat UI from source provided by @rossoskull 
2. Fixed typo in js file as instructed by @aashutoshrathi 

###  Screenshot Dark Mode:
![image](https://user-images.githubusercontent.com/42104907/57173792-cef44780-6e52-11e9-8038-5fe7ceb28d7d.png)


### Screenshot Light Mode:
![image](https://user-images.githubusercontent.com/42104907/57173794-d6b3ec00-6e52-11e9-907b-b32c97984b09.png)


###  Zoomed Dark button:

![image](https://user-images.githubusercontent.com/42104907/57173820-4fb34380-6e53-11e9-9d8f-924eac504b91.png)


###  Zoomed Light button:

![image](https://user-images.githubusercontent.com/42104907/57173826-6b1e4e80-6e53-11e9-940a-5ce8c4157f51.png)

aashutoshrathi commented:
@rossoskull Can you just confirm the changes once again after conflicts get resolved.

AmanRaj1608 commented:
@rossoskull  Please approve changes if it's correct

aashutoshrathi commented:
@AmanRaj1608 if maintainers merge #73 before this PR, you should be ready with rebasing tools with you.
:wink:

AmanRaj1608 commented:
@aashutoshrathi  Ok 😅😂
> @AmanRaj1608 if maintainers merge #73 before this PR, you should be ready with rebasing tools with you.
> 😉

yashLadha commented:
@AmanRaj1608 Resolve conflicts

AmanRaj1608 commented:
@yashLadha @aashutoshrathi @rossoskull 
Done 🎉
Please review if it again

AmanRaj1608 commented:
Little space added for octocat

Dark -> 
![image](https://user-images.githubusercontent.com/42104907/57197249-b1be9680-6f82-11e9-8320-dd1c085d589a.png)

Light ->
![image](https://user-images.githubusercontent.com/42104907/57197255-c13ddf80-6f82-11e9-91d3-4aca8b7e41bb.png)

AmanRaj1608 commented:
@aashutoshrathi 
Please accept PR if it's done

AmanRaj1608 commented on views/index.html:
```diff
@@ -116,5 +118,9 @@
     <script src="/client.js"></script>
+    <script src="/darkmode.js"></script>
+    <!-- Place this tag in your head or just before your close body tag. -->
+    <script async defer src="https://buttons.github.io/buttons.js"></script>
```
removed

AmanRaj1608 commented:
@aashutoshrathi 
Fixed problem ✔
Was having some issue with GitHub Desktop. Now using Git bash 😅
